content/docs/zero/upgrading.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Upgrading
+description: This page discusses how to upgrade your cluster to use the latest version of Pomerium Zero.
+keywords:
+  [
+    pomerium zero release,
+    zero release,
+    zero releases,
+    pomerium zero releases,
+    upgrade zero,
+    zero version,
+    latest zero version,
+  ]
+pagination_prev: null
+pagination_next: null
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade Pomerium Zero
+
+This page shows you how to upgrade your cluster to use the latest version of Pomerium Zero. Refer to the steps for your deployment environment.
+
+:::note
+
+Review the latest [**Core Upgrade guide**](/docs/deploy/core/upgrading) for release details before you upgrade Pomerium Zero.
+
+:::
+
+## How to upgrade Pomerium Zero
+
+<Tabs>
+<TabItem label="Docker" value="docker">
+
+To update Pomerium Zero in Docker:
+
+1. In your `docker-compose.yaml` file, update the Pomerium Zero image tag to specify the [latest tagged release](https://github.com/pomerium/pomerium/tags):
+
+```yaml
+pomerium:
+  image: pomerium/pomerium:vX.Y.Z
+```
+
+1. Run the following command:
+
+```bash
+$ docker compose up -d
+```
+
+Docker should automatically pull the new image of Pomerium Zero before running the container. If for some reason it doesn't pull the image, you can manually run:
```

Comment:
Here as well, I think it's confusing to talk about an "image of Pomerium Zero" because this is the regular pomerium/pomerium docker image, not something specific to Pomerium Zero.